### DISCUSSION TITLE
Settings for local network access

### DISCUSSION THREAD

**Discussion by beardedfool** [jonaswinkler/paperless-ng#987] [Support]
Trying the ansible install, services are runnning but haven't implemented the nginx yet, just trying to see if I can acccess it from the local network but not sure I understand the interplay of 
PAPERLESS_ALLOWED_HOSTS
PAPERLESS_CORS_ALLOWED_HOSTS

Can someone confirm the combination of settings in paperless.conf that should allow access from anywhere on the network to server.dom:8000 please?

I can also see paperlessng_listen_address &  paperlessng_listen_port but they're not in the configuration settings document. 
Does that mean they shouldn't be used, or just the document is out of date? 
Otherwise where do you set a different port?

What's amazing I've also done something exceptionally 'clever' and I've never seen this before....
There's a windows machine (client.dom) I'm using and I can access the paperless server from http://127.0.0.1:8000 !!
Yet I can't access it from server.dom:8000
It's defintely hitting that server as:
* when I close the paperless-webserver on server.dom I loose access
* I can also see files going on server.dom documents/originals when I upload them. 

Zero idea how that's happening! Any guesses as it's sending me into an existential crisis on everything I thought I knew about networking ;)

For any windows people netsh interface portproxy show all is coming up blank, running as Admin

I was playing with the docker before the ansible install but that was also on the server, not on the windows machine and I've cleared browser caches and rebooted. It's thankfully not happening from another machine on the network.

**Comment by f-ohler** [comment]
paperlessng_listen_address (and port) influences the gunicorn settings
set paperlessng_listen_address to 0.0.0.0 to allow access from anywhere
by default it only allows access from localhost
to restrict it to your network, refer to the gunicorn documentation wrt bind address
